index.css

```diff
@@ -29,8 +30,6 @@ h1{
   color: white;
   border-radius: 6%;
   width: 200px;
-
-
 }
 #game {
```

Comment:
you haven't line up your indentations
all tag groups should be indented the same